AdamsToDoConsoleApplication/JsonFileManagement.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace AdamsToDoConsoleApplication
+{
+    class JsonFileManagement
+    {
+        public static void JsonReadFiles(string path, List<Task> taskCollection)
+        {
+            try
+            {
+                string rawFileString = System.IO.File.ReadAllText(path);//System.IO.File.ReadAllText(@"C:\Users\AdamHoskinson\Documents\document.json");
+                JObject loadedObject = JObject.Parse(rawFileString);
+                foreach (var obj in loadedObject["Tasks"])
+                {
+                    var task = new Task(obj["mTitle"].ToString(), obj["mDescription"].ToString(), Convert.ToBoolean(obj["mCompletion"]));
```

Comment:
Again im not sure how this is brought in but does it need ToString. I feel like its already a string again. Intellisense helps for things like this just hover over and see.